lib/bone.js

```diff
@@ -1077,9 +1077,6 @@ class Bone {
    * @return {Spell}
    */
   static create(values, opts = {}) {
-    if (opts.validate !== false) {
-      this._validateAttributes(values);
-    }
```

Comment:
just a second thought, how about still validate here, but turn validation off in `instance.create(values, { validate: false })` L1085

Reply:
that will make validator can't access `this`:
```js

Model.init({
  name: {
     validate: {
        custom: function () {
          return  this.name.length > 10; // this.name will throw error
        }
     }
  }
})
```